src/communicator.ts

```diff
@@ -1,10 +1,10 @@
 import { Span } from "@opentelemetry/sdk-trace-base";
 import { WinstonLogger as Logger } from "./telemetry/logs";
 import { WorkflowContextImpl } from "./workflow";
-import { OperonContext, OperonContextImpl } from "./context";
+import { DBOSContext, DBOSContextImpl } from "./context";
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
-export type OperonCommunicator<T extends any[], R> = (ctxt: CommunicatorContext, ...args: T) => Promise<R>;
+export type DBOSCommunicator<T extends any[], R> = (ctxt: CommunicatorContext, ...args: T) => Promise<R>;
```

Comment:
I'm still debating between `DBOSCommunicator` vs `Communicator`. Same thing for `Workflow` and `Transaction`.